connection.js

```diff
@@ -15,6 +15,13 @@ import {
 
 const logger = require('jitsi-meet-logger').getLogger(__filename);
 
+/**
+ * The feature announced so we can distinguish jibri participants.
+ *
+ * @type {string}
+ */
+export const DISCO_JIBRI_FEATURE = 'http://jitsi.org/protocol/jibri';
```

Comment:
Https?

Reply:
Well I copy-pasted from here: https://github.com/jitsi/jicofo/blob/master/src/main/java/org/jitsi/jicofo/Participant.java#L226
And all others are http, http://jitsi.org/meet/lipsync, http://jitsi.org/jitmeet/audio,  http://jitsi.org/jitmeet ... Even those from the xeps are like this: http://jabber.org/protocol/xhtml-im, http://www.w3.org/1999/xhtml, http://jabber.org/protocol/muc it is just a namespace, do we want to change or it doesn't matter.

Reply:
It’s ok, no problemo then!